src/framework/preferences/js/SeparatedPanelPrefsEditor.js

```diff
@@ -333,4 +333,84 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
         setTimeout(callback, 1);
     };
 
+    fluid.defaults("fluid.prefs.separatedPanel.lazyLoad", {
+        events: {
+            onLazyLoad: null,
+            onPrefsEditorMessagesPreloaded: null,
+            onCreateSlidingPanelReady: {
+                events: {
+                    onPrefsEditorMessagesLoaded: "onPrefsEditorMessagesPreloaded"
+                }
+            },
+            templatesAndIframeReady: {
+                events: {
+                    onLazyLoad: "onLazyLoad"
+                }
+            }
+        },
+        components: {
+            templateLoader: {
+                createOnEvent: "onLazyLoad"
+            },
+            messageLoader: {
+                options: {
+                    events: {
+                        onResourcesPreloaded: "{separatedPanel}.events.onPrefsEditorMessagesPreloaded"
+                    },
+                    preloadResources: "prefsEditor",
+                    listeners: {
+                        "onCreate.loadResources": {
+                            listener: "fluid.prefs.separatedPanel.lazyLoad.preloadResources",
+                            args: ["{that}", {expander: {func: "{that}.resolveResources"}}, "{that}.options.preloadResources"]
+                        },
+                        "{separatedPanel}.events.onLazyLoad": {
+                            listener: "fluid.resourceLoader.loadResources",
+                            args: ["{messageLoader}", {expander: {func: "{messageLoader}.resolveResources"}}]
+                        }
+                    }
+                }
+            },
+            slidingPanel: {
+                options: {
+                    invokers: {
+                        operateShow: {
+                            funcName: "fluid.prefs.separatedPanel.lazyLoad.showPanel",
+                            args: ["{separatedPanel}", "{that}.events.afterPanelShow.fire"],
+                            // override default implementation
+                            "this": null,
```

Comment:
This usability bug should have been fixed by https://issues.fluidproject.org/browse/FLUID-5184 with https://github.com/fluid-project/infusion/blob/master/src/framework/core/js/Fluid.js#L2479 - could you verify that this works fine with these "null" entries removed?